src/sweets/core.py

```diff
@@ -176,7 +176,7 @@ def _set_bbox_and_wkt(self, values):
         else:
             # otherwise, make WKT just a 5 point polygon
             self.wkt = wkt.dumps(geometry.box(*self.bbox))
-        return values
+        return self
```

Comment:
just curious, what was the warning here?

Reply:
```
<home>/open_source/sweets/src/sweets/core.py:218: UserWarning: A custom validator is returning a value other than `self`.
  Returning anything other than `self` from a top level model validator isn't supported when validating via `__init__`.
  See the `model_validator` docs (https://docs.pydantic.dev/latest/concepts/validators/#model-validators) for more details.
```

After validators should return the object itself. The `values` method argument is something else.